client/blocks/comments/post-comment-content.jsx

```diff
@@ -21,21 +38,25 @@ export default class PostCommentContent extends React.Component {
 				</div>
 			);
 		}
-
 		/*eslint-disable react/no-danger*/
 		return (
 			<AutoDirection>
-				<div
-					className="comments__comment-content"
-					dangerouslySetInnerHTML={ { __html: this.props.content } }
-				/>
+				<div className={ classNames( 'comments__comment-content-wrapper', this.props.className ) }>
+					<div
+						className="comments__comment-content"
+						ref="overflowNode"
```

Comment:
this method of setting a ref is considered "legacy" as described in the [React Docs](https://facebook.github.io/react/docs/refs-and-the-dom.html#legacy-api-string-refs). 

Lets avoid it!